src/assets/styles/app.scss

```diff
@@ -0,0 +1,5 @@
+
+@import '../../../node_modules/react-blur-admin/src/assets/styles/blur/theme/conf/colorScheme/_blur';
```

Comment:
I think we may want to import this from react blur admin instead of react webpack skeleton. 

In React Blur Admin in app.scss, we should import all the SCSS files that are needed for the components / pages to run properly. 

Thoughts?